src/selectors/getAllProjectKeys.js

```diff
@@ -3,8 +3,15 @@ import {createSelector} from 'reselect';
 export default createSelector(
   state => state.get('projects'),
   projects => projects.
-    filter(project => project.updatedAt).
-    sortBy(project => -project.updatedAt).
+    sort((firstProj, secondProj) => {
```

Comment:
We can make this a bit terser by destructuring here

Reply:
Can you be more specific?